src/app/TideMatchSet.cpp

```diff
@@ -162,7 +162,7 @@ void TideMatchSet::writeToFile(
 
   bool brief = Params::GetBool("brief-output");
 
-  const Peptide* peptide = peptides->GetPeptide(0);
+  Peptide* peptide = peptides->GetPeptide(0);
```

Comment:
why did you remove the const? I don't think you would be modifying the peptide at all.